app/policies/sample_policy.rb

```diff
@@ -34,4 +34,37 @@ def destroy_attachment? # rubocop:disable Metrics/AbcSize
         )
       )
   end
+
+  scope_for :relation, :exportable_namespace_samples do |relation, options| # rubocop:disable Metrics/BlockLength
+    namespace = options[:namespace]
+    sample_ids = options[:sample_ids]
+
```

Comment:
**nitpick (blocking):** Can you add a `next relation.none unless Member.can_export_data?(user, namespace)` that way if the user can not export samples in the namespace they get an empty list.